tembo-operator/src/app_service/ingress.rs

```diff
@@ -168,6 +200,11 @@ pub fn generate_ingress_routes(
             for route in routings.iter() {
                 match route.ingress_path.clone() {
                     Some(path) => {
+                        if route.entry_points.clone()?.contains(&"ferretdb".to_string()) {
```

Comment:
If the entry point contains "ferretdb" then the ingress route will not be generated.